integration-tests/soak/soak_runner_test.go

```diff
@@ -198,6 +194,6 @@ func soakTestHelper(
 		})).
 		Run()
 	require.NoError(t, err, "Error launching test environment")
-	err = actions.TriggerRemoteTest(exeFile, testEnvironment)
+	err = actions.TriggerRemoteTest("../../", testEnvironment)
```

Comment:
Not in love with this, but also not sure how to improve it at the moment.

Reply:
Ya this is purely caused by our current structure basically being a sub module and not at the root of the repo